docs/src/componentDocs/DrawerSubheader/examples/DrawerSubheaderExample.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  Drawer,
+  DrawerBody,
+  DrawerHeader,
+  DrawerNavGroup,
+  DrawerNavItem,
+  DrawerSubheader,
+} from '@brightlayer-ui/react-native-components';
+import {ExampleShowcase} from '../../../shared';
+import {Text} from 'react-native-paper';
+
+export const DrawerSubheaderExample = (): JSX.Element => (
+  <ExampleShowcase>
+    <Drawer style={{width: 250, margin: 'auto'}}>
+      <DrawerHeader title="Title" />
```

Comment:
why header not picking up default theme.colors.surface. Currently its white background

Reply:
As we discussed in the pre-demo meeting, this should get checked/fixed in the RN component library. Thanks